tmt/base.py

```diff
@@ -776,6 +786,18 @@ def plans(self):
         """ Test plans for execution """
         if self._plans is None:
             self._plans = self.tree.plans(run=self)
+            # Use a default plan with fmf discover when no plan detected
+            if not self._plans:
+                plan = tmt.utils.yaml_to_dict("""
+                    /plans/default:
+                        discover:
+                            how: fmf
```

Comment:
I meant this, maybe we should document clearly what is the default plan ...

Reply:
Good point. Added a user story and examples.